qa/sc_evm_cert_key_rotation.py

```diff
@@ -47,13 +47,17 @@
     - End the WE and verify that the certificates is added to the MC and SC.
```

Comment:
I think we can avoid to add additional key rotation in the test, we just need to modify an existing call by removing the gasLimit.